src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/LibraryImportGenerator.cs

```diff
@@ -445,30 +445,8 @@ private static MemberDeclarationSyntax PrintForwarderStub(ContainingSyntax userD
                     Token(SyntaxKind.ExternKeyword),
                     Token(SyntaxKind.UnsafeKeyword))
                 .WithSemicolonToken(Token(SyntaxKind.SemicolonToken))
-                .WithAttributeLists(
-                    SingletonList(AttributeList(
-                        SingletonSeparatedList(
-                                Attribute(
-                                    NameSyntaxes.DllImportAttribute,
-                                    AttributeArgumentList(
-                                        SeparatedList(
-                                            new[]
-                                            {
-                                                AttributeArgument(LiteralExpression(
-                                                        SyntaxKind.StringLiteralExpression,
-                                                        Literal(libraryImportData.ModuleName))),
-                                                AttributeArgument(
-                                                    NameEquals(nameof(DllImportAttribute.EntryPoint)),
-                                                    null,
-                                                    LiteralExpression(
-                                                        SyntaxKind.StringLiteralExpression,
-                                                        Literal(libraryImportData.EntryPoint ?? stubMethodName))),
-                                                AttributeArgument(
-                                                    NameEquals(nameof(DllImportAttribute.ExactSpelling)),
-                                                    null,
-                                                    LiteralExpression(SyntaxKind.TrueLiteralExpression))
-                                            }
-                                            )))))))
+                .WithAttributeLists(SingletonList(AttributeList(SingletonSeparatedList(
+                    CreateDllImportAttribute(libraryImportData, isForwarder: false, stubMethodName)))))
```

Comment:
Instead of passing optional arguments, could we make a new instance of `libraryImportData` using with-expressions to set the right settings? And then also comment here about why we're setting particular settings (setting the entry point name and removing SetLastError).